src/java.base/share/classes/sun/security/ec/ECPrivateKeyImpl.java

```diff
@@ -247,36 +247,4 @@ private void readObject(ObjectInputStream stream)
         throw new InvalidObjectException(
                 "ECPrivateKeyImpl keys are not directly deserializable");
     }
-
-    /**
-     * Returns the native EC public key context pointer.
-     * @return the native EC public key context pointer or -1 on error
-     */
-    long getNativePtr() {
-        if (this.nativeECKey == 0x0) {
```

Comment:
Please remove the (now) unused field `nativeECKey`.

Reply:
Removed.